activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -23,6 +23,20 @@ def table_alias_for(table_name)
         table_name[0...table_alias_length].tr('.', '_')
       end
 
+      # Returns the relations names usuable to back Active Record models.
+      # Defaults to all #tables and #views.
```

Comment:
plus in Changelog

> These methods determine what relations can be used to back Active Record models (usually tables and views).

Do we intend that user can overwrite `data_sources` when they would not use views as Active Record back?
If so, senny's comment is reasonable.